lib/simplecov.rb

```diff
@@ -158,6 +156,13 @@ def usable?
         false
       end
     end
+
+    #
+    # Clear out the previously cached .result. Primarily useful in testing
+    #
+    def clear_result!
```

Comment:
Imo there should only be `!` versions of methods that exist without `!` to show slightly more dangerous ones so I'd opt to go without this :)